nginx_configs/albs.conf

```diff
@@ -58,6 +58,10 @@ server {
     }
 
     location /pulp/ {
+        # header for decompressing build/test logs
+        if ($request_uri ~* (\.gz\.log$)) {
```

Comment:
Usually `.gz` extension is at the end of the file name, could you please adjust (I assume it will be modification on upload part?) ?